### PR TITLE
Added commands for dateTime support

### PR DESCRIPTION
**Pull request by paveljandejsek**
Solves #28 

Open points - NextMonth and NextCalendarDay commands - do we want datetime alternatives?
Do we want to add shortcodes for datetime parse and format date time similar to date versions? E.g.
``` 
shorthands.put("DP", "DATE_PARSE");
shorthands.put("FD", "FORMAT_DATE");
```

### CODE REVIEW

File: jbehave-support-core/docs/Expression-commands.md [hunk 1]
```diff
@@ -10,10 +10,13 @@ List of commands:
 - [BYTES](#bytes)
 - [CONCAT](#concat)
 - [CURRENT_DATE](#current_date)
+- [CURRENT_DATE_TIME](#current_date)
```
Comment (thradec) [inline jbehave-support-core/docs/Expression-commands.md]: typo, should be `#current_date_time`

File: jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/expression/temporal/CurrentDateCommand.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package org.jbehavesupport.core.internal.expression;
+package org.jbehavesupport.core.internal.expression.temporal;
```
Comment (thradec) [inline jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/expression/temporal/CurrentDateCommand.java]: well, moving old classes into the new package is potentially breaking change

File: jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/expression/temporal/DateParseCommand.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package org.jbehavesupport.core.internal.expression;
+package org.jbehavesupport.core.internal.expression.temporal;
```
Comment (thradec) [inline jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/expression/temporal/DateParseCommand.java]: same question as for `CurrentDateCommand`

File: jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/expression/temporal/RandomDateTimeCommand.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+import org.jbehavesupport.core.expression.ExpressionCommand;
+import org.jbehavesupport.core.internal.RandomGeneratorHelper;
+import org.springframework.stereotype.Component;
+
+/**
+ * Generate random {@link java.time.LocalDateTime}
+ */
+@Component
+public class RandomDateTimeCommand implements ExpressionCommand {
```
Comment (thradec) [inline jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/expression/temporal/RandomDateTimeCommand.java]: it should be mentioned in Expression-commands.md

**Comment by thradec** [comment]
my opinion to your questions

> NextMonth and NextCalendarDay commands - do we want datetime alternatives?

no, until somebody will request it

> Do we want to add shortcodes

no, they will make readability worse

File: jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/expression/temporal/CurrentDateCommand.java (hunk 2 above)
Comment (thradec) [reply jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/expression/temporal/CurrentDateCommand.java]: oops, sorry I overlooked the internal package, then that change is absolutely fine

File: jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/expression/temporal/RandomDateTimeCommand.java (hunk 4 above)
Comment (paveljandejsek) [reply jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/expression/temporal/RandomDateTimeCommand.java]: Added, thanks for noticing.

File: jbehave-support-core/docs/Expression-commands.md (hunk 1 above)
Comment (paveljandejsek) [reply jbehave-support-core/docs/Expression-commands.md]: Fixed

**Pull request merged**